container-search/src/main/java/com/yahoo/prelude/statistics/StatisticsSearcher.java

```diff
@@ -152,11 +152,12 @@ private void qps(Metric.Context metricContext) {
         peakQpsReporter.countQuery();
     }
 
-    private Metric.Context getChainMetricContext(String chainName) {
+    private Metric.Context getChainMetricContext(String chainName, String endpoint) {
         Metric.Context context = chainContexts.get(chainName);
```

Comment:
the cache is accessed by chain only, but now the content varies with both chain and endpoint names, this might be wrong.

Reply:
Thanks @arnej27959! I will look into that